Wrappers/Python/test/test_PluginsAstra_Projectors.py

```diff
@@ -70,6 +71,43 @@ def setUp(self):
 
         self.norm = 14.85
 
+
+    def test_ProjectionOperator_img_geom_default(self):
+        K = ProjectionOperator(image_geometry=None, acquisition_geometry=self.ag, device='gpu')
+        assert(K.volume_geometry == self.ag.get_ImageGeometry())
+
+    def test_ProjectionOperator_acq_geom_default(self):
+        with self.assertRaises(TypeError):
+            ProjectionOperator(image_geometry=self.ig, acquisition_geometry=None, device='gpu')
+    
+    def test_ProjectionOperator_all_default(self):
+        with self.assertRaises(TypeError):
+            ProjectionOperator(image_geometry=None, acquisition_geometry=None, device='gpu')
+    
+    @parametrize("device, no_error_raised, err_type", 
```

Comment:
I think this would be clearer if you flip the logic.
 so "raise_error" still with `True`/`False`, but the if statement will flip too.
 
I think each set of parameters should be on a new line to make to really clear and easy to read.